### ISSUE TITLE
Heading levels and ARIA tags

### ISSUE THREAD

**Issue by mariongalley** [ThreeSixtyGiving/dataquality#144]
Repro steps: Run the DQT through WAVE and AXE devtools in Firefox to diagnose the following issues:

**Missing H1 and skipped heading levels (WAVE)**

![Image](https://github.com/user-attachments/assets/e5c61b0d-e645-4781-a18e-bc47f5863119)

**ARIA labels missing children (AXE devtools)**

![Image](https://github.com/user-attachments/assets/f56c65d9-e0f6-47c0-9826-0b91ead2a25e)

These could be design system level issues
